### DISCUSSION TITLE
Since the last version (8.9.2 from april 30 2021) the "sendlater" column in drafts i empty

### DISCUSSION THREAD

**Discussion by 1337-TIM** [Extended-Thunder/send-later#324] [Q&A]
I wrote on the issue

**Comment by jph-sendlater** [comment]
It would help if I could get a debug log from this. Please try opening the error console (Ctrl+Shift+J) and then switch folders back and forth a few times. Let me know what you see in the console.

**Reply by 1337-TIM** [reply]
HI
Thank you so much !
THis is what I get in the non empty columns (see images below)
<img width="1045" alt="debug-col" src="https://user-images.githubusercontent.com/17205394/117110464-6f550d80-ad86-11eb-8b1b-7bfa46abd060.png">
<img width="1048" alt="information-col" src="https://user-images.githubusercontent.com/17205394/117110470-711ed100-ad86-11eb-9aa2-506fc3e0adbe.png">
<img width="1053" alt="journal-col" src="https://user-images.githubusercontent.com/17205394/117110476-71b76780-ad86-11eb-98a6-2576a923259a.png">
<img width="1046" alt="errors-col" src="https://user-images.githubusercontent.com/17205394/117110480-7419c180-ad86-11eb-8d36-35ab9d920d1c.png">

**Reply by 1337-TIM** [reply]
Well well ...  always the same version (8.9.2 from April 30), same version of thunderbird but yesterday I applied the new "big sur" update and ... the column appears again.

So if you are on "big sur" as me, the update of the os could help. (again, I cleared the PRAM before, and nothing changed ... justthis new version of big sur yesterday)
For other OS if it happens, I can't say.

For me, this is closed : thank you for this fantastic add-on ! I just regret I can't use GPG with it but it's another question 

Merci pour tout et excusez mon anglais ;)

**Reply by 1337-TIM** [reply]
euh ... sorry but it doesn't work again ?????
Waht did i do ? nothing special , just looking at my mail and "send later" another. And when I turn back to my draft folder ... empty again 

So what ? I don't know. FOrtunatly, there's the "send it later" button on each message but I have to click to each one to see it

**Reply by jph-sendlater** [reply]
Unfortunately none of the errors in your console are related to send later, so that turns out to not be much help. Someone opened this as an issue: #325 so I'll respond there with more details.

**Comment by jikamens** [comment]
The column doesn't exist anymore because it's not possible to implement it in TB115, and when it does become possible to implement it again it'll be all new code, so there's no advantage to leaving this discussion open.
